From 24dffdc2410627966a82d7cc8b29797cecbab61b Mon Sep 17 00:00:00 2001
From: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
Date: Mon, 31 Jan 2022 11:02:36 +0300
Subject: [PATCH 222/249] test/jenkins: test for old mount engine

Let's run zdtm in jenkins with --mntns-compat-mode option and same for
device-external mount test from others.

Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
---
 test/jenkins/criu-compat-mounts.sh | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100755 test/jenkins/criu-compat-mounts.sh

diff --git a/test/jenkins/criu-compat-mounts.sh b/test/jenkins/criu-compat-mounts.sh
new file mode 100755
index 000000000..cc68035f3
--- /dev/null
+++ b/test/jenkins/criu-compat-mounts.sh
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Make one regular C/R cycle with mount-v2 disabled
+set -e
+source `dirname $0`/criu-lib.sh
+prep
+FAIL=0
+./test/zdtm.py run --all --mntns-compat-mode --keep-going --report report --parallel 4 || FAIL=$?
+
+# Make device-external mounts test
+EXTRA_OPTS=--mntns-compat-mode make -C test/others/mnt-ext-dev/ run || FAIL=$?
+
+if [ $FAIL -ne 0 ]; then
+	fail
+fi
-- 
2.35.1

